docs/source/mturk.rst

```diff
@@ -0,0 +1,181 @@
+..
```

Comment:
made it a bit verbose because I felt that the mturk part is not as self-explanatory as the rest of the platform.. also I believe @alexholdenmiller is working on another tutorial?